docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,452 @@
+# Initialize Vendor Accounts
+
+This series of documents explains how to create a new instance of the DANDI project (i.e. a clone of the entire DANDI ecosystem). It is suggested that you briefly read through each of the
+documents in this guide before starting.
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**
+
+• **AWS**
+
+• **GitHub**
+
+• **Terraform Cloud**
+
+• **Netlify**
+
+• **Sentry**
+
+• **PyPI**
+
+• **Datalad (TBD)**
+
+• **git-annex (TBD)**
+
+## Heroku
+
+##### Create your own Heroku account
+
+No special steps here, just create!
+
+##### Create a "Team" 
+
+Send invites to the appropriate e-mail addresses to onboard your team
+
+##### Create an "App"
+
+<br/><br/>
+<img
+src="../img/heroku_create_app.png"
```

Comment:
I believe the `img` directory is within `docs`.  Will need to change the path for all images.

```suggestion
src="./img/heroku_create_app.png"
```

Reply:
Hmmm changing this broke all the image references -- I think it is working as intended (unless you are observing something different locally?)

Reply:
Thanks for checking.  The references are broken in the GitHub UI, but I didn't test locally with MkDocs.  Perhaps the working directory for MkDocs is different so it works out.  Let's ignore this suggestion.